MapboxAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java

```diff
@@ -1687,6 +1705,10 @@ private boolean handleTwoFingersTap(MotionEvent event) {
     @Override
     public boolean onTouchEvent(MotionEvent event) {
 //        Log.i(TAG, "onTouchEvent with event = " + event);
```

Comment:
OK, cool. Heads up though, the commented out code is from [upstream](github.com/americanredcross/openmapkitandroid)